### PR TITLE
Provide MeasurementGrp as a parameter in ct macro

### PR DESCRIPTION
**Pull request by droldan**
Add measurement group as a optional parameter using the Option parameter feature

### CODE REVIEW

File: src/sardana/macroserver/macros/standard.py [hunk 1]
```diff
@@ -41,6 +41,8 @@
     ViewOption, iMacro, Hookable
 from sardana.macroserver.msexception import StopException
 from sardana.macroserver.scan.scandata import Record
+from sardana.macroserver.msparameter import Optional
```
Comment (reszelaz) [inline src/sardana/macroserver/macros/standard.py]: The idea is to imprort it from sardana.macroserver.macro as in the example.

File: src/sardana/macroserver/macros/standard.py [hunk 2]
```diff
@@ -648,17 +650,24 @@ class ct(Macro, Hookable):
     env = ('ActiveMntGrp',)
     hints = {'allowsHooks': ('pre-acq', 'post-acq')}
     param_def = [
-        ['integ_time', Type.Float, 1.0, 'Integration time']
+        ['integ_time', Type.Float, 1.0, 'Integration time'],
+        ['mnt_grp', Type.MeasurementGroup, Optional, 'MntGrp to use']
```
Comment (reszelaz) [inline src/sardana/macroserver/macros/standard.py]: I would not use accronyms in the description.

File: src/sardana/macroserver/macros/standard.py (hunk 1 above)
Comment (droldan) [reply src/sardana/macroserver/macros/standard.py]: Provided in [43eabb4](https://github.com/sardana-org/sardana/pull/940/commits/43eabb45148e4ea0512a146f7b7cb7577bf0b237) commit.

File: src/sardana/macroserver/macros/standard.py (hunk 2 above)
Comment (droldan) [reply src/sardana/macroserver/macros/standard.py]: Provided in [43eabb4](https://github.com/sardana-org/sardana/pull/940/commits/43eabb45148e4ea0512a146f7b7cb7577bf0b237) commit.

**Comment by reszelaz** [comment]
Great! Thanks for the changes. Merging..

**Pull request merged**